### PR TITLE
Unit Tests for Script class

### PR DESCRIPTION
**Pull request by igormcoelho**
Let's get 100% coverage :muscle:

### CODE REVIEW

File: src/neo-vm/Script.cs [hunk 1]
```diff
@@ -9,6 +9,28 @@ public class Script
         private readonly byte[] _value;
         private readonly ICrypto _crypto;
 
+        /// <summary>
```
Comment (shargon) [inline src/neo-vm/Script.cs]: I prefer the constructor after the properties

**Comment by erikzhang** [comment]
Wait for #120

File: src/neo-vm/Script.cs (hunk 1 above)
Comment (igormcoelho) [reply src/neo-vm/Script.cs]: These C# guys...... xD

**Comment by vncoelho** [comment]
@igormcoelho, PR120 has been already merged.

**Comment by igormcoelho** [comment]
Nice! I'll finish the tests and pass it on.

**Comment by shargon** [comment]
Any update?

**Comment by shargon** [comment]
conflicts

**Pull request closed without merging**